crates/ide-completion/src/completions/extern_crate.rs

```diff
@@ -0,0 +1,96 @@
+//! Completion for extern crates
+
+use ide_db::{FxHashSet, SymbolKind};
+use syntax::{
+    ast::{self, NameRef},
+    AstNode,
+};
+
+use crate::{context::CompletionContext, CompletionItem, CompletionItemKind};
+
+use super::Completions;
```

Comment:
nit, we prefer not to use `super` imports, I assume this was copied from another file (we have a few rogue super imports lying around still)